tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Builders/OrganisationBuilder.cs

```diff
@@ -66,9 +68,21 @@ internal OrganisationBuilder WithAddress(Address value)
             return this;
         }
 
+        internal OrganisationBuilder WithRelatedOrganisation(Guid relatedOrganisationId)
+        {
+            relatedOrganisation = new()
```

Comment:
I think it might be slightly clearer to include the type here (but think `new()` is fine elsewhere, e.g. if we were doing something like `return new()` here).